commit a1ab67287c8d1285e9a36f1974d7c999a56d1c48 (refs/changes/85/5185/1)
Author: Jordan Paige Hendricks <jordan.hendricks@joyent.com>
Date:   2018-12-04T00:47:49+00:00 (10 months ago)
    
    OS-7066 Want support ahci LED entries in topo (fix DEBUG)

diff --git a/usr/src/lib/fm/topo/modules/common/fac_prov_ahci/fac_prov_ahci.c b/usr/src/lib/fm/topo/modules/common/fac_prov_ahci/fac_prov_ahci.c
index 719444df0e..cd47bc6a23 100644
--- a/usr/src/lib/fm/topo/modules/common/fac_prov_ahci/fac_prov_ahci.c
+++ b/usr/src/lib/fm/topo/modules/common/fac_prov_ahci/fac_prov_ahci.c
@@ -94,7 +94,6 @@ do_led_control(topo_mod_t *mod, const char *devctl, uint32_t port,
 
 	if (set) {
 		ahci_ioc_em_set_t ahci_set;
-		ahci_em_led_state_t led_state;
 		uint32_t led_status = *ledmode;
 		uint_t op, leds;
 
